docs/get-started-without-a-framework.md

```diff
@@ -0,0 +1,118 @@
+---
+id: get-started-without-a-framework
+title: Get Started Without a Framework
+hide_table_of_contents: true
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import constants from '@site/core/TabsConstants';
+
+import RemoveGlobalCLI from './\_remove-global-cli.md';
+
+If you have constraints that are not served well by a Framework, or you prefer to write your own Framework, you can create a React Native app without using a Framework.
+
+To do so, you'll first need to [set up your environment](set-up-your-environment). Once you're set up, continue with the steps below to create an application and start developing.
+
+### Step 1: Creating a new application
+
+<RemoveGlobalCLI />
+
+You can use [React Native Community CLI](https://github.com/react-native-community/cli) to generate a new project. Let's create a new React Native project called "AwesomeProject":
```

Comment:
Can we reuse here `<PlatformSupport />` component which will mention which platforms React Native Community CLI support? -> macOS, Windows, visionOS etc. that as of my knowledge any other Framework doesn't support.